### PR TITLE
removed failed css props, used scrollview, fix type issue

### PR DESCRIPTION
**Pull request by hdJerry**
Image

Fixed.

- removed failed css props, used scrollview, fix type issue

### CODE REVIEW

**Review by AllStackDev1 — APPROVED** [review]

File: expo-zustand-styled-components/src/components/IssuesTabView/IssuesTabView.styles.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ type ScreenWidth = {
   screenWidth: number | undefined;
 };
 
-export const MainContainer = styled.View<ScreenWidth>`
+export const MainContainer = styled.ScrollView<ScreenWidth>`
   width: ${({ screenWidth }) => (screenWidth > breakpoints.tablet ? '80%' : '100%')};
   margin-horizontal: auto;
   margin-top: 40px;
```
Comment (AllStackDev1) [inline expo-zustand-styled-components/src/components/IssuesTabView/IssuesTabView.styles.ts:12]: I will suggest here padding-top to margin-top
```suggestion
  padding-top: 32px;
```

File: expo-zustand-styled-components/src/components/PullRequestsTabView/PullRequestsTabView.styles.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ type ScreenWidth = {
   screenWidth: number | undefined;
 };
 
-export const MainContainer = styled.View<ScreenWidth>`
+export const MainContainer = styled.ScrollView<ScreenWidth>`
   width: ${({ screenWidth }) => (screenWidth > breakpoints.tablet ? '80%' : '100%')};
   margin-horizontal: auto;
   margin-top: 40px;
```
Comment (AllStackDev1) [inline expo-zustand-styled-components/src/components/PullRequestsTabView/PullRequestsTabView.styles.ts:12]:
```suggestion
  padding-top: 32px;
```

**Review by AllStackDev1 — CHANGES_REQUESTED** [review]
Those CSS changes i suggested will make the padding space scroll with the rest of the elements.

File: expo-zustand-styled-components/src/components/IssuesTabView/IssuesTabView.styles.ts (hunk 1 above)
Comment (hdJerry) [reply expo-zustand-styled-components/src/components/IssuesTabView/IssuesTabView.styles.ts:12]: Why padding?

File: expo-zustand-styled-components/src/components/PullRequestsTabView/PullRequestsTabView.styles.ts (hunk 2 above)
Comment (hdJerry) [reply expo-zustand-styled-components/src/components/PullRequestsTabView/PullRequestsTabView.styles.ts:12]: The padding adds to the element's width or height, so I am wondering why padding?

Comment (AllStackDev1) [reply expo-zustand-styled-components/src/components/PullRequestsTabView/PullRequestsTabView.styles.ts:12]: With Margin-top u can notice the it added a space that doesn't scroll alone with the content, making it appear weird

https://user-images.githubusercontent.com/36667083/228395061-0bdf16cd-8450-438d-9632-1bdc87b0a6cd.mp4

with padding-top, the space is still there but scrolls with the contents


https://user-images.githubusercontent.com/36667083/228395214-9570fb95-3eca-440d-a918-775017ac164c.mp4

**Review by AllStackDev1 — APPROVED** [review]

**Pull request merged**